web/src/components/storage/DevicesManager.js

```diff
@@ -138,11 +138,15 @@ export default class DevicesManager {
    * Disk devices and LVM volume groups used for the installation.
    * @method
    *
-   * @note The used devices are extracted from the actions.
+   * @note The used devices are extracted from the actions, but the optional argument
```

Comment:
Note: DeviceManager is one of a few files that still pending to be migrated to TypeScript. Just saying :wink:

Reply:
Look! It's a three-headed monkey!